ws/protomsg.go

```diff
@@ -105,3 +110,31 @@ type Error struct {
 	// error occurs it is passed for reference in the Body of the error message
 	MessageID string `msgpack:"msgid,omitempty" json:"message_id,omitempty"`
 }
+
+// ProtoMsg handshake semantics:
```

Comment:
when you presented the versioning, what came to my mind then as an alternative solution was:
1. UI connects to `/api/management/v${version}/deviceconnect/devices/${deviceId}/connect`
2. mender-connect connects to `/api/devices/v${version}/deviceconnect/connect`
3. deviceconnect propagates the `${version}` in the `ProtoMsg.ProtoHdr.Properties["version"]`
@alfrunes and @all I am putting it here to have your comments and more for the record. Versioning is a nice addition.

Reply:
This would definitely solve the current issue. However, I think we should keep the "ProtoMsg" versioning detached from the REST version so that we can negotiate versions independent of the HTTP deviceconnect API. Imagine a future case where the device initiates a handshake (e.g. with the deviceconnect backend), then this type of versioning because there would be no version negotiation. Another drawback is that we cannot actually update the REST version of the API handler without renaming the endpoint.